cli/tests/unit/http_test.ts

```diff
@@ -0,0 +1,126 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+import { assert, assertEquals, deferred, unitTest } from "./test_util.ts";
+import { BufReader, BufWriter } from "../../../test_util/std/io/bufio.ts";
+import { TextProtoReader } from "../../../test_util/std/textproto/mod.ts";
+
+unitTest({ perms: { net: true } }, async function httpServer() {
+  const promise = deferred<void>();
+
+  (async () => {
```

Comment:
We don't need `deferred()` here.
```suggestion
  const promise = (async () => {
```
Ditto for other tests.

Reply:
thanks - fixed